pymatviz/ptable.py

```diff
@@ -397,16 +397,20 @@ def add_elem_symbols(
 
             # Get axis index by element symbol
             symbol: str = element.symbol
+            if symbol not in self.data.index:
```

Comment:
Wondering why this part was even working before? As if we don't check if an element is present, the symbol of all elements would be plotted as in 97d453708e53ec377b78a921aa42f5ffdb7870ed